### ISSUE TITLE
Fail to take over map from c:geo

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/WhereYouGo#250]
Using our current market release 2020.12.03 and WhereYouGo nightly 2021.01.05-NB I am able to take over the c:geo used offline map into WhereYouGo (given it uses Mapsforge v3 format)

However using our current c:geo nightlies it fails to take over the map with:
"Fehler beim Festlegen der Kartendatei (not a file: /storage/emulated/0/offlinemaps)"

@moving-bits @eddiemuc @fm-sys 
Wasn' there a change of how we store the used offline map file / folder recently between 2020.12.03 and today?
It seems this makes the takeover in WhereYouGo fail.

**Comment by eddiemuc** [comment]
Yes, as of now only the Offline Map **Directory** is transferred to WhereYouGo, not the currently selectred file.

This could be fixed. for now. However, latest with the merge of SAF WhereYouGo can no longer use this information from c:geo since
* it will then be an Uri
* it will be calculated "on-the-fly" out of base uri and map name
* it will not be reachable by WhereYouGo unless WhereYouGo implements scoped storage and the user allows access to same dir

This was discussed in #8457 around 25 Nov 2020 and is an unavoidable regression. We have to find a new strategy for WhereYouGo how to deal with offline maps I am afraid, the current hack solution will no longer work.

**Comment by Lineflyer** [comment]
> This was discussed in #8457 around 25 Nov 2020 and is an unavoidable regression. We have to find a new strategy for WhereYouGo how to deal with offline maps I am afraid, the current hack solution will no longer work.

Understand....now it gets a bit complicated:
In fact the user who reported #249 was able to overcome it with the workaround to just get the map file location from c:geo.
However once we remove that function (and we should do so) we need to fix #249 or even better upgrade the mapsforge library for WhereYouGo.

**Comment by SchoolGuy** [comment]
As said in the original issue for mapsforge v4/v5 support... When the settings are done I will implement the new map formats.

**Comment by Lineflyer** [comment]
@SchoolGuy Thanks for working on it! Highly appreciated.

I recently had a user on support wanting to contribute to WhereYouGo. I directed him to the repository..so some help might be on the way.

**Comment by eddiemuc** [comment]
It should be fairly simple to replace the current map file selection dialog in whereyougo with an intent call to action-open-document. If mapsforge 0.3 already supports opening map files via inputstream then this can be done even before map lib upgrade .

**Comment by moving-bits** [comment]
For the time being PR #420 removes the menu entry "acquire map from c:geo", as it currently does not make sense any longer.
Functionality can be reactivated when WhereYouGo starts supporting both SAF and Mapsforge v5 maps.
